Gemfile.lock

```diff
@@ -433,6 +433,7 @@ GEM
     mini_histogram (0.3.1)
     mini_magick (4.11.0)
     mini_mime (1.0.3)
+    mini_portile2 (2.5.1)
```

Comment:
Are the Gemfile.lock changes required? I wonder if this was staged from your local environment, but not strictly needed.

Reply:
Pretty sure it was for Gitpod, didn't mean to commit these, sorry!